tests/0120-asymmetric_subscription.c

```diff
@@ -169,10 +169,14 @@ int main_0120_asymmetric_subscription(int argc, char **argv) {
         rd_kafka_mock_topic_create(mcluster, "t3", _PART_CNT, 1);
         rd_kafka_mock_topic_create(mcluster, "t4", _PART_CNT, 1);
 
-
-        do_test_asymmetric("roundrobin", bootstraps);
-        do_test_asymmetric("range", bootstraps);
-        do_test_asymmetric("cooperative-sticky", bootstraps);
+        /* These tests would be duplicate with KIP 848 protocol,
+         * given we're not testing the assignment here,
+         * as it's mocked. */
+        if (test_consumer_group_protocol_classic()) {
+                do_test_asymmetric("range", bootstraps);
+                do_test_asymmetric("roundrobin", bootstraps);
+                do_test_asymmetric("cooperative-sticky", bootstraps);
+        }
```

Comment:
We are not checking anything with this test now for the new protocol. I think we should enable "cooperative-sticky" test for the new protocol.

Reply:
Before it was the range assignor that was tested, but I removed it, because, as said, we don't need to test the mock implementation unless it's during its development for making client implementation debugging more easy, that's why I added the unit tests for that at the beginning to make sure we tested with a mock implementation without flaws.

If you want I can keep the `range` test as before as the simple mock implementation is range based

Reply:
Done